Extensions/PlatformBehavior/tests/platformerobjectruntimebehavior.spec.js

```diff
@@ -1135,4 +1343,232 @@ describe('gdjs.PlatformerObjectRuntimeBehavior', function () {
       expect(object.getBehavior('auto1').isGrabbingPlatform()).to.be(true);
     });
   });
+
+  describe('(walk)', function () {
+    let runtimeScene;
+    let object;
+    let platform;
+
+    beforeEach(function () {
+      runtimeScene = makeTestRuntimeScene();
+
+      // Put a platformer object on a platform
+      object = new gdjs.RuntimeObject(runtimeScene, {
+        name: 'obj1',
+        type: '',
+        behaviors: [
+          {
+            type: 'PlatformBehavior::PlatformerObjectBehavior',
+            name: 'auto1',
+            gravity: 1500,
+            maxFallingSpeed: 1500,
+            acceleration: 500,
+            deceleration: 1500,
+            maxSpeed: 500,
+            jumpSpeed: 900,
+            canGrabPlatforms: true,
+            ignoreDefaultControls: true,
+            slopeMaxAngle: 60,
+            jumpSustainTime: 0.2,
+            roundCoordinates: true,
+          },
+        ],
+      });
+      object.getWidth = function () {
+        return 10;
+      };
+      object.getHeight = function () {
+        return 20;
+      };
+      runtimeScene.addObject(object);
+
+      // Put a platform.
+      platform = addPlatformObject(runtimeScene);
+      platform.setPosition(0, -10);
+    });
+
+    const fall = (frameCount) => {
+      for (let i = 0; i < frameCount; ++i) {
+        const lastY = object.getY();
+        runtimeScene.renderAndStep(1000 / 60);
+        expect(object.getBehavior('auto1').isFalling()).to.be(true);
+        expect(object.getBehavior('auto1').isMoving()).to.be(true);
+        expect(object.getY()).to.be.above(lastY);
+      }
+    };
+
+    const walkRight = (frameCount) => {
+      for (let i = 0; i < frameCount; ++i) {
+        const lastX = object.getX();
+        object.getBehavior('auto1').simulateRightKey();
+        runtimeScene.renderAndStep(1000 / 60);
+        expect(object.getBehavior('auto1').isOnFloor()).to.be(true);
+        expect(object.getX()).to.be.above(lastX);
+        //TODO probably a bug, uncomment when it's fixed.
+        //expect(object.getBehavior('auto1').isMoving()).to.be(true);
```

Comment:
isMoving is true when you:
- fall
- jump 
- or your speed of your horizontal movement is not 0 *and* you've moved from at least 1 pixel.

The last thing may make the "isMoving" to be false if you've not "moved enough".